### PR TITLE
Updated readme, contribution guidelines and code of conduct

### PR DESCRIPTION
**Pull request by javfg**
Related issue: https://github.com/opentargets/platform/issues/1054.

### CODE REVIEW

**Review by LucaFumis — APPROVED** [review]
Thank you @javfg - looks great!

File: docs/customisations.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+target/
+ sections/
+   ...
+   <section-id>/
+     custom/
+       ... // it may help to put any other components you create here,
+           // to keep them separate from the template/expected ones below
+     Description.js
+     index.js
+     Section.js
```
Comment (fedde-s) [inline docs/customisations.md:17]: Body.js?

Comment (fedde-s) [reply docs/customisations.md:17]: Or is this pull request not up to date with <https://github.com/opentargets/platform/issues/1131#issuecomment-697681546>?

Comment (javfg) [reply docs/customisations.md:17]: Good catch, that I have to remove that part of the section.

**Review by mirandaio — APPROVED** [review]

**Pull request merged**